README.rst

```diff
@@ -1 +1,25 @@
-An extension for use in the L{twisted.internet.iocpreactor} I/O Completion Ports reactor.
+twisted-iocpsupport
+===================
+
+.. image:: https://img.shields.io/github/workflow/status/twisted/twisted-iocpsupport/Build%20and%20upload%20to%20PyPI?label=GitHub%20Actions&logo=github
+    :alt: GitHub Actions
+    :target: https://github.com/twisted/twisted-iocpsupport/
+.. image:: https://img.shields.io/pypi/v/twisted-iocpsupport?logo=pypi
+    :alt: PyPI
+    :target: https://pypi.org/project/twisted-iocpsupport/
+
+
+An extension for use in the L{twisted.internet.iocpreactor} I/O Completion
+Ports reactor.
+
+This code was initially part of the core Twisted project. It was moved into a
```

Comment:
Maybe start with a brief summary and then explain all the details.

```
**This is a protected package. It should not be used outside of Twisted project.**
```

Reply:
```suggestion
**This is a protected package. It should not be used outside of Twisted project.**
This code was initially part of the core Twisted project. It was moved into a
```

Reply:
hmm I think it's clear enough for a v1.0.0, we can always do a v1.0.0.post0 for readme changes